frontend/src/components/VHeader/VHeaderMobile/VHeaderMobile.vue

```diff
@@ -263,11 +329,24 @@ export default defineComponent({
       }
       if (([keycodes.Escape] as string[]).includes(key)) {
         // Hide the recent searches.
-        isRecentSearchesModalOpen.value = false
+        closeRecentSearchesModal()
+      }
+      if (key === keycodes.Tab) {
+        // Keep `focused` as true when the user tabs out of the input.
+        isInputFocused.value = false
```

Comment:
This comment seems incongruous with the logic below, is it describing something else?